src/Css.elm

```diff
@@ -3236,6 +3242,58 @@ slashedZero =
   }
 
 
+{- FEATURE TAG VALUES -}
+
+
+{-| Alias for `on` value of [`font-feature-settings`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-feature-settings#Formal_syntax)
+-}
+on : Int
+on =
```

Comment:
It also says "The two keywords on and off are synonyms for 1 and 0 respectively" so I thought it might be okay just to leave them out. If y'all think we should include them how about `featureOn` and `featureOff`?